app/src/main/java/io/bankingofthings/iot/repo/QrRepo.kt

```diff
@@ -0,0 +1,38 @@
+package io.bankingofthings.iot.repo
+
+import android.graphics.Bitmap
+import android.os.Environment
+import io.bankingofthings.iot.storage.SpHelper
+import io.bankingofthings.iot.utils.QRUtil
+import java.io.File
+import java.io.FileOutputStream
+
+class QrRepo(private val spHelper:SpHelper, private val deviceRepo: DeviceRepo) {
+    val qrBitmap: Bitmap = QRUtil.createBitmap(deviceRepo.deviceModel)
+
+    init {
+        // Do only for every new device
+        if (!spHelper.getHasStoredQrImage()) {
```

Comment:
shouldn't comment `// Do only for every new device` be a documentation for the class?